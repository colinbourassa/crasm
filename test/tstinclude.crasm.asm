;;; Author: Leon Bottou
;;; Public Domain.

	ilist on
	include tstmacros.crasm.asm
	ilist off
	include tstif.crasm.asm
