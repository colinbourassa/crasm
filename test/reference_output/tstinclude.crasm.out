S1133000AD001000040003000200010000300B308A
S1133010093007300530036573706F69720001650C
S11330206E636F726500040003000200010000304B
S11330302D302B30293027302500020000100000ED
S1133040002000000003030303030303030303033B
S1133050030303030303030303030303030303033C
S1133060030303030303030303030303030303032C
S1133070030303030303030303030303030303031C
S1133080030303030303030303030303030303030C
S113309003030303030303030303030303030303FC
S11330A00303030303030303036A65206D27696EA7
S11330B07465726573736520656E636F72652061F4
S11330C0752063616C63756C2065742061206C27C6
S11330D061666669636861676520646573207472FC
S10830E06565732E007C
S11320001FFC400000AD0FA003E80FA013880FA031
S9030000FC
